ui/components/app/assets/hooks/useMultichainAssets.tsx

```diff
@@ -0,0 +1,70 @@
+import { useSelector } from 'react-redux';
+import { Hex } from '@metamask/utils';
+import { getMultichainBalances } from '../../../../selectors/multichain';
+import {
+  getAccountAssets,
+  getAssetsMetadata,
+} from '../../../../selectors/assets';
+import { ParsedAssetId, parseAssetId } from '../util/parseAssetId';
+import { CHAIN_ID_TOKEN_IMAGE_MAP } from '../../../../../shared/constants/network';
+
+const accountId = 'e0bcfad0-2d3f-48cd-b50f-8a2f1d4f7dea';
+
+const useMultiChainAssets = () => {
+  const multichainBalances = useSelector(getMultichainBalances);
+  const accountAssets = useSelector(getAccountAssets);
+  const assetsMetadata = useSelector(getAssetsMetadata);
+
+  const assetIds = accountAssets[accountId] || [];
```

Comment:
We should check quickly if `[]` will trigger a re-render since it's a new object.  If so, we can create a constant out of scope that will always be the same object.